core/domain/user_services.py

```diff
@@ -359,7 +359,7 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
-                role=feconf.ROLE_ID_ADMIN,
+                role=feconf.ROLE_ID_EXPLORATION_EDITOR,
```

Comment:
ping @seanlip ... please reply to this. Other than this, I think I have addressed rest of the comments.